decanter_ai_sdk/client.py

```diff
@@ -520,6 +520,14 @@ def predict_ts(
         )
         return prediction
 
+    def stop_uploading(self, id):
```

Comment:
give a type hint for the return type. Maybe it should be a boolean about whether the stop was successful or not

Reply:
I'm thinking maybe I can use `res.ok` to detect the response of the server? and if it's ok I can just log the success message from the server, if it's not then I can log some error message ?